modules/imgui/src/guimemorycomponent.cpp

```diff
@@ -65,11 +66,16 @@ void GuiMemoryComponent::render() {
     _isEnabled = v;
     _isCollapsed = ImGui::IsWindowCollapsed();
 
+    uint64_t ram = global::openSpaceEngine->ramInUse();
+    ImGui::Text("RAM: %u (%.2f MiB)", ram, ram / (1024.f * 1024.f));
```

Comment:
Who's our intended audience for this?  Mebibytes are seemingly quite rarely used as a unit 😅 

And depending who we expect to use this feature, we should probably consider what unit we use. I'm personally fine with whichever

Reply:
Also, I think we can skip the `.2f` here, considering the size and context of the numbers? :)

Reply:
Removed the .2f.   I'll leave the MiB since that is the correct unit for dividing by 1024 ;)   And also I see this ImGui panel only being used by Devs;  in general a regular user shouldn't need to look at this